docs/reference/hacl/dh/ec/k256.md

```diff
@@ -0,0 +1,71 @@
+<!-- Note: This module is well-documented in hacl-star. Blocked by https://github.com/cryspen/hacl-star/issues/14 -->
+
+# K-256
+
+## Verified field arithmetic modulo p = 2^256 - 0x1000003D1.
```

Comment:
Let's not call things verified here. Reading this I'd think "oh, so all the other code isn't verified".